coalib/settings/SectionFilling.py

```diff
@@ -29,12 +30,20 @@ def fill_settings(sections, acquire_settings, log_printer):
 
     for section_name, section in sections.items():
         bear_dirs = section.bear_dirs()
-        bears = list(section.get('bears', ''))
-        section_local_bears, section_global_bears = collect_bears(
-            bear_dirs,
-            bears,
-            [BEAR_KIND.LOCAL, BEAR_KIND.GLOBAL],
-            log_printer)
+        if hasattr(section, 'aspects') and section.aspects is not None:
```

Comment:
`getattr(section, 'aspects', None)` is one way to simplify this.

but IMO `section.aspects` should be set to `None` if `aspects` are not being used.